### ISSUE TITLE
Image Imports

### ISSUE THREAD

**Issue by justnealpatel** [HappenApps/Quiver#562]
I'm not sure if it's just me, but every time I drag an image or import one, it always comes out zoomed in and more blurred. Is there a fix to this? My original image is not blurred but when I import it into Quiver, it becomes blurred.

**Comment by ylian** [comment]
What's the original image's format? I think TIFFs are converted to JPEGs so that might be the issue. Other image formats are kept as they are. TIFFs are converted because of browser compatibility issues, but I guess I could convert them to PNG instead.

Not sure what you mean by "zoomed in". Is it clipped?

**Comment by justnealpatel** [comment]
It's a .png. When I usually import photos, I usually take screenshots and import them. I tried playing around with importing different sizes and it seems that some larger sizes are being blurred a bit. I think I fixed my problem though. By zoomed in, I meant the picture appeared to become less sharp (not by much though but still noticeable). Is there any possibility of a scaling feature to images?
